### DISCUSSION TITLE
RealBasicVSR模型训练效果如何优化？

### DISCUSSION THREAD

**Discussion by yunping-zhu** [open-mmlab/mmediting#1282] [How-to]
你好，我基于mmedit工具，使用REDS数据集， 训练了RealBasicVSR x2的模型。按照官方文档，先训练了realbasicvsr_wogan_x2的模型，然后训练realbasicvsr_x2微调模型，但是微调模型效果不太好，处理出来的视频还是有伪影，训练的时候迭代到70000次的时候PSNR值就开始逐渐降低了，请教大牛有没有办法改善这个效果？我的训练配置有没有改善的地方？下面是我的训练日志。

realbasic_wogan_x2模型日志：
[20220610_230000.log](https://github.com/open-mmlab/mmediting/files/8937146/20220610_230000.log)

realbasic_x2 模型日志：
[20220614_211438.log](https://github.com/open-mmlab/mmediting/files/8937107/20220614_211438.log)
[20220616_103657.log](https://github.com/open-mmlab/mmediting/files/8937108/20220616_103657.log)

其中微调模型训练的loss呈现如下的变化趋势：
<img width="1034" alt="loss" src="https://user-images.githubusercontent.com/53961673/174516059-30d060b4-08b8-4de0-8a12-b062c7f148c9.png">

**Comment by mm-assistant** [comment]
We recommend using English or English & Chinese for issues so that we could have broader discussion.

**Comment by mayunjie** [comment]
how to set x2 model

**Comment by zengyh1900** [comment]
@Z-Fran 
Please check this issue.

**Comment by zhu2bowen** [comment]
@yunping-zhu 请问你的问题解决了吗？我也想训练个x2的模型，遇到类似问题。

**Comment by Feynman1999** [comment]
In theory, the effect should not be much worse than the official x4 model? can you share the result?
